operator/node.go

```diff
@@ -78,15 +78,15 @@ func New(opts Options) Node {
 
 // Start starts to stream duties and run IBFT instances
 func (n *operatorNode) Start() error {
-	n.logger.Info("starting node -> IBFT")
+	n.logger.Info("All required services are ready. OPERATOR SUCCESSFULLY CONFIGURED AND NOW RUNNING!")
```

Comment:
`RUNNING` --> `STARTING` would be more correct..

Reply:
at that point the node is already after sync and all the necessary preparations so it is basically running